rules/prefer-to-contain.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const getDocsUrl = require('./util').getDocsUrl;
+const expectCase = require('./util').expectCase;
+const expectNotCase = require('./util').expectNotCase;
+const expectResolveCase = require('./util').expectResolveCase;
+const expectRejectCase = require('./util').expectRejectCase;
+const method = require('./util').method;
+const argument = require('./util').argument;
+
+module.exports = {
+  meta: {
+    docs: {
+      url: getDocsUrl(__filename),
+    },
+    fixable: 'code',
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          !(
+            expectNotCase(node) ||
```

Comment:
what about e.g. `expect(a.includes(b)).not.toEqual(true);`? might be weird enough that we don't need to support it, I guess 🙂

Reply:
It'a a little weird but it's a nice to have. Added it to the rule!